platform/core/src/services/MeasurementService/MeasurementService.ts

```diff
@@ -83,6 +83,20 @@ const VALUE_TYPES = {
   ROI_THRESHOLD_MANUAL: 'value_type::roiThresholdManual',
 };
 
+// Contains priority levels for the jump to image.  This allows firing the
+// jump to image several times with increasing priority to apply the change
+// to different sets of viewports until some viewport handles it.
+const JUMP_TO_MEASUREMENT = {
+  // The grid layout should be updated to include this
+  GRID_PRIORITY: 2,
```

Comment:
I would recommend this

```js
const JUMP_PRIORITY = {
  ALL_VIEWPROTS: 'allviewports', 
  ANY_VIEWPORT: 'anyviewport',
  SELECTED_VIEWPORT: 'selectedpriority'
}

Reply:
The issue is that it needs to be numeric because it needs to increase, so one can't just name them - they are inherently ordered.